/**:
  ros__parameters:
    dlp_path: '/dlp-dataset/data/DJI_0012' # Starts from home directory